extensions/wikia/ArticleVideo/bower_components/html5-skin/js/controller.js

```diff
@@ -704,6 +785,16 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.renderSkin();
     },
 
+    onShowAdTimeLeft: function (event, showAdTimeLeft) {
+      this.state.showAdTimeLeft = showAdTimeLeft;
+      this.renderSkin();
+    },
+
+    onShowAdFullScreenToggle: function (event, showAdFullScreenToggle) {
+      this.state.showAdFullScreenToggle = showAdFullScreenToggle;
+      this.renderSkin();
+    },
+
     onSkipAdClicked: function(event) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "event". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)